notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -1645,6 +1645,58 @@ human nature [It was human nature to romanticize war, according to the malevolen
+human nature [A message of the film is that a loss of humanity comes with increased intelligence.]
+apotheosis [The story culminated with Lucy transcending space and time upon reaching 100% usage of her brain capacity, and becoming one with the universe or some such thing.]
+
+:: Minor Themes
+boyfriend and girlfriend [Lucy's new boyfriend, Richard, got her tangled up with a drug smuggling ring.]
+the theory of evolution [Professor Norman lectured the evolution of the nervous system.]
+the control of fire [A caveman was shown starting a fire by means of repeatedly hitting two stones together over a pile of brush.]
+smuggling [Lucy was forced to transport a bag of drugs, which was sewn into her abdomen, from Taiwan to Europe.]
+academia [Professor Norman gave a university lecture on the subject of animals, humans included, using only a fraction of their total brain capacity.]
+super hearing [Lucy acquired the ability to hear people talking well outside of an ordinary person's earshot.]
+mother and daughter [An unanesthetized Lucy spoke to her mother over the phone as a surgeon excised a bag of illicit drugs from her abdomen.]
+super intelligence [Lucy acquired this ability in spades after an experimental, illicit drug got into her bloodstream. She learned Chinese in an hour, how to drive in an instant, became an expert in neuroscience in a single night, etc.]
+the desire for vengeance [Lucy avenged Mr. Jang forcing her to serve as his drug mule by grabbing him firmly by the balls for awhile.]
+mind reading ability [Lucy read from Mr. Jang's mind the locations of the other drug mules.]
+chemicals [Exposure to the CPH4 synthesized chemical endowed Lucy with heightened physical and mental abilities.]
+adapting to life in a foreign culture [Lucy was studying in Taipei without knowing the language.]
+law enforcement [The police captain Pierre Del Rio led a search for a trio of drug mules.]
+what it is like on a passenger flight [Lucy made something of a scene on an otherwise routine passenger flight to Paris.]
+telekinetic ability [Lucy acquired this power which she used to, among other things, make the bullets fall out of a gun that was being pointed at her.]
+shapeshifting [Lucy acquired the ability to transform her body at will. For instance, she changed the color of her hair and later changed her hand into a completely different hand.]
+time travel [Lucy ultimately acquired an innate ability to travel in time, which she used to go back to see the dinosaurs and the Big Bang itself.]
+
+:: Other Keywords
+electromagnetic wave manipulation ability [Lucy acquired an ability to manipulate radio and other electromagnetic waves, enabling her to make herself appear on peoples computer screens at will.]
```

Comment:
https://www.themeontology.org/theme.php?name=electricity%20manipulation%20ability ?

Reply:
I also though we had a theme about controlling machines in general but can't find it